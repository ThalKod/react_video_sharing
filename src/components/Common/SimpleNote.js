import React from "react";

export default function SimpleNote({ text}){
  return (
      <div className="content-wrapper">
        <div className="container">
          <div className="row">
            <div className="col-lg-12 upload-page">
              <div className="u-area">
                {text}
              </div>
            </div>
          </div>
        </div>
      </div>
      )
};
