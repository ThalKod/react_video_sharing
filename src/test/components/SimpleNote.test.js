import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json"

import SimpleNote from "components/Common/SimpleNote";

describe("SimpleNote Components", () => {
  it("should render the components correctly", () => {
    const wrapper = shallow(<SimpleNote text="test"/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
