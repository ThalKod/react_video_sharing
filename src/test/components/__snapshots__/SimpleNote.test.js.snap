// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleNote Components should render the components correctly 1`] = `
<div
  className="content-wrapper"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-lg-12 upload-page"
      >
        <div
          className="u-area"
        >
          test
        </div>
      </div>
    </div>
  </div>
</div>
`;
